frontend/src/pages/settings/Settings.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  BaseTemplate,
+  Body,
+  Column,
+  Form,
+  FormField,
+  Grid,
+  H1,
+  Modal,
+  P,
+  SerializedFormData,
+} from "@maykin-ui/admin-ui";
+import { FormEvent, Fragment, useState } from "react";
+import { useLoaderData } from "react-router-dom";
+
+import { useSubmitAction } from "../../hooks";
+import "./Settings.css";
+import { SettingsSection, SettingsSectionProps } from "./components";
+import { UpdateSettingsAction } from "./settings.action";
+import { SettingsContext } from "./settings.loader";
+
+export type SettingsPageProps = React.ComponentProps<"main"> & {
+  // Props here.
+};
+
+/**
+ * Settings page
+ */
+export function SettingsPage({ children, ...props }: SettingsPageProps) {
+  const { zaaktypesShortProcess, zaaktypeChoices } =
+    useLoaderData() as SettingsContext;
+  const submitAction = useSubmitAction<UpdateSettingsAction>();
+
+  const [modalOpenState, setModalOpenState] = useState<boolean>(false);
+
+  const verkorteProcedureFormFields: FormField[] = [
+    {
+      autoComplete: "off",
+      label: "Verkorte zaaktypes",
+      name: "verkorte_zaaktypes",
+      type: "checkbox",
+      options: zaaktypeChoices.map((choice) => ({
+        value: choice.value,
+        label: choice.label,
+        defaultChecked: zaaktypesShortProcess.includes(String(choice.value)),
+      })),
+    },
+  ];
+
+  const handleSubmitPatchArchiveConfig = async (
+    _: FormEvent,
+    data: SerializedFormData,
+  ) => {
+    const verkorteZaaktypes = (data.verkorte_zaaktypes ?? []) as string[];
+    submitAction({
+      type: "PATCH-ARCHIVE-CONFIG",
+      payload: {
+        zaaktypesShortProcess: verkorteZaaktypes,
+      },
+    });
+    setModalOpenState(true);
+  };
+
+  const settingsArray: SettingsSectionProps[] = [
+    {
+      settingTitle: "Verkorte procedure",
+      settingDescription:
+        "Pas hier de zaaktypes aan die in aanmerking komen voor de verkorte procedure.",
+      children: (
+        <Form
+          fields={verkorteProcedureFormFields}
+          validateOnChange={true}
+          onSubmit={handleSubmitPatchArchiveConfig}
+        />
+      ),
+    },
+  ];
+
+  return (
+    <BaseTemplate>
```

Comment:
I would suggest simplifying this, I wrote down some steps to end up with a cleaner layout:

- Replace BaseTemplate with CardBaseTemplate
- Replace H1 with H2
- Remove second HR
- Remove additional grids and columns
- Remove SettingsSection, and fragment, render setting.children directly (is this data structure really needed, can’t this be a regular Form)?
- 
![Uploading Scherm­afbeelding 2024-07-18 om 17.54.34.png…]()